src/Equinox/Decider.fs

```diff
@@ -108,6 +108,39 @@ type Decider<'event, 'state>(inner : DeciderCore<'event, 'state>) =
+    /// 2. (if events yielded) Attempt to sync the yielded events to the stream.
+    ///    (Restarts up to <c>maxAttempts</c> times with updated state per attempt, throwing <c>MaxResyncsExhaustedException</c> on failure of final attempt.)
+    /// 3. Uses <c>render</c> to generate a 'view from the persisted final state
+    member _.TransactTask(interpret : 'state -> CancellationToken -> Task<'event list>, render : 'state -> 'view, ?load, ?attempts, ?ct) : Task<'view> =  task {
+        let inline interpret' s ct = task { let! es = interpret s ct in return Seq.ofList es }
+        return! inner.TransactAsync(interpret', render, ?load = load, ?attempts = attempts, ?ct = ct) }
+
+    /// 1. Invoke the supplied <c>Task</c> returning <c>decide</c> function with the present state, holding the <c>'result</c>
+    /// 2. (if events yielded) Attempt to sync the yielded events to the stream.
+    ///    (Restarts up to <c>maxAttempts</c> times with updated state per attempt, throwing <c>MaxResyncsExhaustedException</c> on failure of final attempt.)
+    /// 3. Yield result
+    member _.TransactTask(decide : 'state -> CancellationToken -> Task<'result * 'event list>, ?load, ?attempts, ?ct) : Task<'result> =  task {
+        let inline decide' s ct = task { let! r, es = decide s ct in return struct (r, Seq.ofList es) }
+        return! inner.TransactAsync(decide = decide', ?load = load, ?attempts = attempts, ?ct = ct) }
+
+    /// 1. Invoke the supplied <c>Task</c> returning <c>decide</c> function with the current complete context, holding the <c>'result</c>
+    /// 2. (if events yielded) Attempt to sync the yielded events to the stream.
+    ///    (Restarts up to <c>maxAttempts</c> times with updated state per attempt, throwing <c>MaxResyncsExhaustedException</c> on failure of final attempt.)
+    /// 3. Yield result
+    member _.TransactExTask(decide : ISyncContext<'state> -> CancellationToken -> Task<'result * 'event list>, ?load, ?attempts, ?ct) : Task<'result> =  task {
+        let decide' c ct = task { let! r, es = decide c ct in return struct (r, Seq.ofList es) }
+        return! inner.TransactExAsync(decide = decide', ?load = load, ?attempts = attempts, ?ct = ct) }
+
+    /// 1. Invoke the supplied <c>Task</c> returning <c>decide</c> function with the current complete context, holding the <c>'result</c>
+    /// 2. (if events yielded) Attempt to sync the yielded events to the stream.
+    ///    (Restarts up to <c>maxAttempts</c> times with updated state per attempt, throwing <c>MaxResyncsExhaustedException</c> on failure of final attempt.)
+    /// 3. Yields a final 'view produced by <c>mapResult</c> from the <c>'result</c> and/or the final persisted <c>ISyncContext</c>
+    member _.TransactExTask(decide : ISyncContext<'state> -> CancellationToken -> Task<'result * 'event list>, mapResult : 'result -> ISyncContext<'state> -> 'view,
+                             ?load, ?attempts, ?ct) : Task<'view> =  task {
```

Comment:
remove 2 spaces ;)

Reply:
😂, even my fancy vim indent config can't handle your demands!

Reply:
there's a pile of `. =` sequences, can you clean them too while you're a it please?